ngraph/src/ngraph/op/convolution.cpp

```diff
@@ -26,7 +26,7 @@ using namespace std;
 using namespace ngraph;
 
 // *** Convolution OP SET 1 ***
-constexpr NodeTypeInfo op::v1::Convolution::type_info;
+RTTI_DEFINITION("Convolution", op::v1::Convolution, Node, 1)
```

Comment:
Parent class is Op.

Reply:
Op is useless class and I haven't create an extra level in hierarchy for it. Each level in hierarchy increases time of the type identification so it is not designed to introduce levels unless really necessary.